README.md

```diff
@@ -1,4 +1,4 @@
-[![licenseimg]][licenselink]  [![Codacy][codacyimg]][codacylink] [![Build Status](https://travis-ci.com/SimonJPegg/mvn_scalafmt.svg?branch=master)](https://travis-ci.com/SimonJPegg/mvn_scalafmt)
+[![licenseimg]][licenselink]  [![Codacy][codacyimg]][codacylink] ![Build Status][BuildImg]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-image] Use the trailing [] on reference images](https://app.codacy.com/manual/Antipathy_org/mvn_scalafmt/pullRequest?prid=5511915)